appservice/src/createAppService/AppServicePlanListStep.ts

```diff
@@ -35,24 +35,23 @@ export class AppServicePlanListStep extends AzureWizardPromptStep<IAppServiceWiz
         );
     }
 
-    public async prompt(wizardContext: IAppServiceWizardContext): Promise<IAppServiceWizardContext> {
-        if (!wizardContext.plan && !wizardContext.newPlanName) {
```

Comment:
consider reusing shouldPrompt

Reply:
ditto on others

Reply:
Reusing how? I _deleted_ `if (!wizardContext.plan && !wizardContext.newPlanName) {`

Reply:
Huh...  Wonder what my mental state was this morning.